Unable to install

By these instructions: https://github.com/cirruslabs/cirrus-cli/blob/2f6cd75/INSTALL.md#golang

```
> go get github.com/cirruslabs/cirrus-cli/...
# github.com/go-git/go-git/plumbing/transport/ssh
../go/src/github.com/go-git/go-git/plumbing/transport/ssh/common.go:147:15: undefined: proxy.Dial
# google.golang.org/grpc/internal/transport
../go/src/google.golang.org/grpc/internal/transport/http_util.go:603:6: f.fr.SetReuseFrames undefined (type *http2.Framer has no field or method SetReuseFrames)

> ls $GOPATH/bin/cirrus*
fish: No matches for wildcard “$GOPATH/bin/cirrus*”.

// there are many other binaries

> go version
go version go1.15.1 linux/amd64
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -28,10 +28,10 @@ curl -L -o cirrus https://github.com/cirruslabs/cirrus-cli/releases/latest/downl
 
 ## Golang
 
-If you have [Go](https://golang.org/) installed, you can run:
+If you have [Go](https://golang.org/) 1.15 or later installed, you can run:
 
 ```
-go get github.com/cirruslabs/cirrus-cli/...
+(cd && GO111MODULE=on go get github.com/cirruslabs/cirrus-cli/...)
 ```
 
 This will build and place the `cirrus` binary in `$GOPATH/bin`.

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -28,7 +28,7 @@ curl -L -o cirrus https://github.com/cirruslabs/cirrus-cli/releases/latest/downl
 
 ## Golang
 
-If you have [Go](https://golang.org/) 1.15 or later installed, you can run:
+If you have [Go](https://golang.org/) 1.15 or newer installed, you can run:
 
 ```
 (cd && GO111MODULE=on go get github.com/cirruslabs/cirrus-cli/...)

